views/term_table.erb

```diff
@@ -1,9 +1,19 @@
 <div class="page-section" id="term">
-    <div class="container">
-        <h1 class="text-center"><span class="avatar"><i class="fa fa-university"></i></span> <%= @term[:name] %></h1>
+    <div class="container text-center">
+        <h1><span class="avatar"><i class="fa fa-university"></i></span> <%= @term[:name] %></h1>
+
       <% unless @term[:start_date].to_s.empty? and @term[:end_date].to_s.empty? %>
-        <p class="text-center"><%= @term[:start_date] %> – <%= @term[:end_date] or 'current' %></p>
+        <p><%= @term[:start_date] %> – <%= @term[:end_date] or 'current' %></p>
+      <% end %>
+
+        <a href="<%= @urls[:csv] %>" class="term-download">
```

Comment:
Presumably this was meant to be a link to the JSON rather than the CSV? Either way, I think even having this link here at all is confusing for not really much value, especially now that you can download the full JSON from the country page. I think I'll just remove it…